src/app/features/fss-search/fss-simplified-search/fss-simplified-search.component.scss

```diff
@@ -28,4 +28,27 @@
         flex-grow: 1;
 		margin-left: 70px;
     }
-  }
\ No newline at end of file
+}
+.switchToESS{
+	width: 100%; 
+	float: right;
+    text-align: right;
+	padding: 0px;
+}
+	.Exchangesetpara{
+		font-size: 17px;
+	}
+
+  .Searchmainaindiv{
```

Comment:
I think there's a typo in the name of the `.Searchmainaindiv` class. Should it be `.Searchmaindiv` (`.searchMainDiv`)?

Reply:
Resolved by doing changes